### PR TITLE
config: check the validity of the DDL lease

### PR DESCRIPTION
**Pull request by zimulala**
### What problem does this PR solve? 
In general, there are multiple TiDB instances in a cluster. Prevents schema information from being out of sync when there are multiple instances. Except for our test environment(we will use "mocktikv"), we cannot set DDL lease to 0s.

### What is changed and how it works?
If we use the storage of `tikv`, we cann't set the value of "DDL lease" to 0s.

### Check List 

Tests 

 - Local

Related changes

 - Need to cherry-pick to the release branch

Release note

 - Check the validity of the DDL lease.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -756,6 +756,9 @@ func (c *Config) Valid() error {
 	if c.Store == "mocktikv" && !c.RunDDL {
 		return fmt.Errorf("can't disable DDL on mocktikv")
 	}
+	if c.Store == "tikv" && (c.Lease == "0s" || c.Lease == "0" || c.Lease == "-0s" || c.Lease == "-0") {
```
Comment (wjhuang2016) [inline config/config.go]: How about `0.0s`?

**Comment by zimulala** [comment]
PTAL @bb7133

**Comment by zimulala** [comment]
PTAL @wjhuang2016 @crazycs520

File: config/config.go [hunk 2]
```diff
@@ -729,6 +729,15 @@ func (c *Config) Load(confFile string) error {
 	return err
 }
 
+// MinDDLLease returns the minimum valid value of the DDL lease.
+func (c *Config) MinDDLLease() time.Duration {
+	minDDLLease := time.Duration(0)
```
Comment (wjhuang2016) [inline config/config.go]: The logic of this function can be simplified?
if c.Store == "tikv" {
return time.Second
}
return time.Duration(0)

**Pull request merged**